types/storybook__addon-knobs/index.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for @storybook/addon-knobs 3.0
+// Project: https://github.com/storybooks/storybook
+// Definitions by: Joscha Feth <https://github.com/joscha>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/// <reference types="storybook__react" />
+
+// TODO: Once TS2.3 is released,
+// https://github.com/Microsoft/TypeScript/issues/14819 should be fixed.
+// Then, upgrade this package's typescript version to 2.3 and
```

Comment:
The branch with the `declare` removals diffs as following to this one here: https://github.com/joscha/DefinitelyTyped/compare/joscha/add-@storybook/addon-knobs...joscha:joscha/add-@storybook/addon-knobs-remove-declare?expand=1